src/content/api/rate-limits.mdoc

```diff
@@ -0,0 +1,36 @@
+---
+title: Rate Limits
+description: Introduction to Rate Limits
+nav:
+  path: API Reference
+  order: 8
+---
+In order to maintain availability to our services by protecting against rapid requests, Centrapay uses Rate Limits to restrict how many requests can be made within a given time period.
+
+At Centrapay rate limits are defined by **Requests Per Minute (rpm).** 
+
+The general rate limit is 200rpm, however for certain APIs we may apply a specific lower or higher rate limit in order to match the expected usage of the endpoint.
+
+Rate limits for authenticated endpoints are applied to the account making the call. This means if you are using a `X-Centrapay-Account` header, the rate limits will still be consumed for the account that the jwt or api key belongs to, not the targeted account.
+
+If you require higher rate limits for your use case, please contact support to request an increase.
```

Comment:
Change: Everywhere that we mention to contact support, we should specify the actual email address - integrations@centrapay.com